### DISCUSSION TITLE
About .vsconfig

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#8075] [General]
Once again creating a new project template, I paid attention to the [.vsconfig file](https://github.com/unoplatform/uno/blob/master/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-net6/.vsconfig).
There seem to be some redundant elements here, but I would like to consult before offering a PR:
- `Microsoft.VisualStudio.Component.Wcf.Tooling` - Seems very specific to be set by all users, as far as I know not needed for normal use of Uno
- `Microsoft.VisualStudio.Component.PortableLibrary` - As far as I know PCL is deprecated and not needed
- `16 Azure components` - This is about 5 GB of additional space after installation. I have never installed it and have been using Uno for the last year with no problems. 

This may make it easier to deploy for some users, but there is one point - I tried Azure when I first started to get acquainted with Uno. It costs about $50 per month for each published application, although Uno can be published on any static site host. Should Azure be recommended for beginners?

**Comment by jeromelaban** [comment]
It seems there are two questions here. Wcf tooling is indeed likely not needed, we could remove it from the list, same for portable library.

As for azure components, those may be related to some of the ASP.NET tooling for publishing.

Finally, [Azure Static Apps are free](https://platform.uno/docs/articles/guides/azure-static-webapps.html?q=static%20apps) and are good for beginners.

**Reply by HavenDV** [reply]
> Finally, [Azure Static Apps are free](https://platform.uno/docs/articles/guides/azure-static-webapps.html?q=static%20apps) and are good for beginners.

Thanks for the link, didn't know about that.

> It seems there are two questions here. Wcf tooling is indeed likely not needed, we could remove it from the list, same for portable library.

I prepared a small PR. It looks pretty trivial, but I need to start somewhere: https://github.com/unoplatform/uno/pull/8093

**Reply by HavenDV** [reply]
I noticed another problem - unoapp-winui-net6 generates a UWP project, which is then not used in the solution.
